src/core/discord-bot/service/command.service.js

```diff
@@ -0,0 +1,20 @@
+import { COMMAND_PREFIX } from 'core/common/constant';
+import { COMMAND_OF } from 'core/common/enum/bot-command';
+import { DiscordSlideService } from '../slide/service';
+
+export class DiscordCommandService {
+    constructor(msg) {
+        this.msg = msg.content;
+        this.commandType = this.msg.slice(0, this.msg.indexOf('_'));
+        this.discordSlideService = DiscordSlideService;
```

Comment:
Because this class is not singleton so I think you shouldn't inject DiscordSlideService into this, just use it directly